cmd/kubent/main.go

```diff
@@ -119,18 +119,8 @@ func main() {
 		log.Fatal().Err(err).Str("name", "Rego").Msg("Failed to evaluate input")
 	}
 
-	var prnt printer.Printer
-	switch config.Output {
-	case "json":
-		prnt, err = printer.NewJSONPrinter(&printer.JSONOpts{})
-	default:
-		prnt, err = printer.NewTextPrinter(&printer.TextOpts{})
-	}
-	if err != nil {
-		log.Fatal().Err(err).Str("name", config.Output).Msg("Failed to initialize output printer")
-	}
-
-	err = prnt.Print(results)
+	printer, _ := printer.ParsePrinter(config.Output)
```

Comment:
We shouldn't be ignoring the error here. Also the variable name is confusing, as this is not a printer, more like `newPrinterFn`.